gems/aws-sdk-s3/lib/aws-sdk-s3/customizations/object.rb

```diff
@@ -188,10 +188,11 @@ def presigned_post(options = {})
       #
       # @return [String]
       #
-      def presigned_url(http_method, params = {})
+      def presigned_url(method, params = {})
         presigner = Presigner.new(client: client)
+        method = "#{http_method}_object" if %w(delete head get put).include?(method)
```

Comment:
These will be symbols, not strings. However `%i` will not work because we support Ruby 1.9.3 for now.

Reply:
I came up with a rather explicit solution, but it goes well in ruby 1.9.3 :sweat_smile: